src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -578,13 +595,13 @@ private SmugglerProgressBase.Counts ProcessRevisionDocuments(SmugglerResult resu
             return result.RevisionDocuments;
         }
 
-        private SmugglerProgressBase.Counts ProcessDocuments(SmugglerResult result, BuildVersionType buildType)
+        private SmugglerProgressBase.Counts ProcessDocuments(SmugglerResult result, BuildVersionType buildType, IEnumerable<DocumentItem> docs = null)
```

Comment:
Move to ProcessDocumentInternal and in ProcessDocuments use previous behavior and use ProcessDocumentInternal in the second place.